.evergreen/preinstall.sh

```diff
@@ -39,7 +39,7 @@ if [ -n "$IS_WINDOWS" ]; then
     rm -rf node-v$NODE_JS_VERSION-win-$ARCH
 
     echo "Installing npm@$NPM_VERSION..."
-    rm -rf npm npx npm.cmd npx.cmd
+    rm -rf npm npx npm.cmd npx.cmd npm.ps1 npx.ps1
```

Comment:
This makes me wonder why we're not using a tool like [fnm](https://github.com/Schniz/fnm) to contain this complexity? 🤔 Is it just that it wasn't around when we originally wrote this?

Reply:
This code is ~7 years old, I wasn't around. Using one would be nice. `mms` uses asdf so that is probably a good choice: https://github.com/asdf-vm/asdf-nodejs 
I started adding that but I would rather do separately in another pr. Right now focusing on getting CI moving again.

Reply:
https://github.com/mongodb-js/compass/pull/6938